proposals/NNNN-Read-Generic-Network-Signal-data.md

```diff
@@ -286,7 +286,7 @@ SDL needs to track requests by NetworkDataIdentifier per app and work through th
 </struct>
 ```
 
-The reason we did not choose this approach is that this will need changing to exiting behavior of GetVehicleData/SubscribeVehicleData/UnsubscribeVehicleData/OnVehicleData. E.g.:
+The reason we did not choose this approach is that this will need changes to the "exiting behavior" of GetVehicleData/SubscribeVehicleData/UnsubscribeVehicleData/OnVehicleData. E.g.:
```

Comment:
This should say "changes to the existing behavior" (fixing "exiting" to "existing" and removing the quotation marks).